src/mesh/mesh_base.C

```diff
@@ -164,6 +166,8 @@ MeshBase& MeshBase::operator= (MeshBase && other_mesh)
   _allow_remote_element_removal = other_mesh.allow_remote_element_removal();
   _block_id_to_name = std::move(other_mesh._block_id_to_name);
   _elem_dims = std::move(other_mesh.elem_dimensions());
+  _elemset_codes = std::move(other_mesh._elemset_codes);
```

Comment:
I *think* this should still work correctly in the pointers case, but I'd love some `#ifdef DEBUG` check below just to be certain.